package.json

```diff
@@ -101,6 +101,7 @@
     "appname": "ros",
     "rosPage": "/ros",
     "oops": "/oops",
-    "noPermissions": "/no-permissions"
+    "noPermissions": "/no-permissions",
+    "rosDetails": "/ros/:systemId"
```

Comment:
You should use inventory detail in here, it will populate the basic information for you, so the URL should be `/ros/:inventoryId`

Reply:
updated to `/ros/:inventoryId`